### PR TITLE
Fix LCD notebook

### PR DESCRIPTION
**Pull request by marcusabate**
Updates the LCD notebook for testing.
Some filenames were changed on the library side, and previously the notebook didn't plot VIO results alongside the VIO+PGO results to show the difference in performance.

### CODE REVIEW

File: notebooks/plot_backend.py [hunk 1]
```diff
@@ -78,8 +78,8 @@
 
 # %%
 # Define directory to VIO output csv files as well as ground truth absolute poses.
-vio_output_dir = "/home/tonirv/datasets/kimera_rig/output_logs/"
-gt_data_file = ""
+vio_output_dir = ""
+gt_data_file = vio_output_dir + "traj_gt.csv"
```
Comment (nathanhhughes) [inline notebooks/plot_backend.py:82]: I'd consider using `pathlib` or `os.path.join` here (makes it less error-prone to remember if you need a trailing slash or not). My preference would be `pathlib`, which is in the standard library in python 3 (which we should consider mandating the use of)

File: notebooks/plot_frontend.py [hunk 2]
```diff
@@ -74,7 +74,7 @@
 # %%
 # Define directory to VIO output csv files as well as ground truth absolute poses.
 vio_output_dir = ""
-gt_data_file = ""
+gt_data_file = vio_output_dir + "traj_gt.csv"
```
Comment (nathanhhughes) [inline notebooks/plot_frontend.py:77]: ditto for `pathlib` or `os.path.join`

File: notebooks/plot_lcd.py [hunk 3]
```diff
@@ -60,101 +73,21 @@
 
 # %%
 # Define directory to VIO output csv files as well as ground truth absolute poses.
-vio_output_dir = "/home/sparklab/code/SparkVIO/output_logs/"
-gt_data_file = "/home/sparklab/datasets/EuRoC/mh_04_difficult/mav0/state_groundtruth_estimate0/data.csv"
-
+vio_output_dir = ""
+gt_data_file = vio_output_dir + "traj_gt.csv"
```
Comment (nathanhhughes) [inline notebooks/plot_lcd.py:77]: ditto for `pathlib` or `os.path.join`

**Review by nathanhhughes — APPROVED** [review]
Looks good to me! If I get a chance, I may try the LCD notebook out, but don't let that hold up merging this.

**Review by lucacarlone — APPROVED** [review]
this looks good to me but did not take a super-close look.

File: notebooks/plot_backend.py (hunk 1 above)
Comment (lucacarlone) [reply notebooks/plot_backend.py:82]: @marcusabate : let me know if you want to do this, otherwise I'll go ahead and merge

**Comment by marcusabate** [comment]
Merging, we should open a PR to change filenames to use `pathlib` across the entire repo.

**Pull request merged**